coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -16,11 +16,18 @@ class IgnoreResultAction(ResultAction):
 
     @staticmethod
     @enforce_signature
-    def is_applicable(result: Result, original_file_dict, file_diff_dict):
+    def is_applicable(result: Result,
+                      original_file_dict,
+                      file_diff_dict,
+                      applied_actions=[]):
```

Comment:
Sure, I can change that. Should I also change the non-default parameter type to a tuple, to be consistent?
In that case should I use a tuple for the `successful_actions` variable, or just convert to tuple when it is passed to the function?